### PR TITLE
Fix conda install on Travis

### PR DESCRIPTION
**Pull request by iskandr**
Also adds pypandoc so that the README renders properly on PyPI

[Review on Reviewable](https://reviewable.io/reviews/hammerlab/varcode/134)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,15 +3,34 @@ language: python
...
+  # We do this conditionally because it saves us some downloading if the
+  # version is the same.
+  - if [[ "$TRAVIS_PYTHON_VERSION" == "2.7" ]]; then
+      wget https://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O miniconda.sh;
+    else
+      wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
+    fi
+  - bash miniconda.sh -b -p $HOME/miniconda
+  - export PATH="$HOME/miniconda/bin:$PATH"
+  - hash -r
```
Comment (arahuja) [inline .travis.yml]: Worth documenting what's happening? OOC, what is this for?

File: pylintrc [hunk 2]
```diff
@@ -1,4 +1,4 @@
 [TYPECHECK]
 # Without ignoring this, we get errors like:
 # E:249,20: Module 'numpy' has no 'nan' member (no-member)
-ignored-modules = numpy
+ignored-modules = numpy
```
Comment (arahuja) [inline pylintrc]: Git seems to want a new line here

**Comment by arahuja** [comment]
LGTM, 2 random notes

File: .travis.yml (hunk 1 above)
Comment (iskandr) [inline .travis.yml]: This resets the lookup table for paths. I'm not sure if that's necessary after resetting the path (I assumed that export PATH=anything should also do `hash -r` under the hood) but didn't feel like experimenting with this already tedious script.

**Pull request merged**